src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.component.js

```diff
@@ -52,6 +55,14 @@ class ExistingPaymentMethodsController {
     }
   }
 
+  addCvvValidators () {
+    this.$scope.$watch('$ctrl.creditCardPaymentForm.securityCode.$viewValue', (number) => {
+      this.creditCardPaymentForm.securityCode.$validators.minLength = cruPayments.creditCard.cvv.validate.minLength
```

Comment:
This is causing an error on localhost. `securityCode` is undefined. This might work staging, butI have nly tested on local.